Yosemite/Yosemite/Stores/JetpackConnectionStore.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+import Networking
+import WordPressKit
+
+/// Handles `JetpackConnectionAction`
+///
+public final class JetpackConnectionStore: DeauthenticatedStore {
+
+    // Keep a strong reference to network to keep requests alive
+    private var network: WordPressOrgNetwork?
```

Comment:
❓  I hope that `Remote` will [hold a reference](https://github.com/woocommerce/woocommerce-ios/blob/trunk/Networking/Networking/Remote/Remote.swift#L11) to `Network`. And we are holding a strong reference to `JetpackConnectionRemote` down below. 
Do we still have to hold a strong reference to `WordPressOrgNetwork`?

Reply:
You are right - there's no need to keep the reference to `network` here. Some of our other stores seem to keep strong references to both `network` and `remote` as `network` is a requirement for `Store`. Do you think this can cause retained cycles?

Anyway - I updated `DeauthenticatedStore` following this suggestion in 0a61d25a33c157d2870ca0eb6d9dddeab16575b5.